api/v1/serializers/members-serializer.js

```diff
@@ -0,0 +1,86 @@
+
+const appRoot = require('app-root-path');
+const decamelize = require('decamelize');
+const JsonApiSerializer = require('jsonapi-serializer').Serializer;
+const _ = require('lodash');
+
+const { serializerOptions } = appRoot.require('utils/jsonapi');
+const { openapi } = appRoot.require('utils/load-openapi');
+const { apiBaseUrl, resourcePathLink, paramsLink } = appRoot.require('utils/uri-builder');
+
+const memberResourceProp = openapi.definitions.MemberResource.properties;
+const memberResourceType = memberResourceProp.type.enum[0];
+const memberResourceKeys = _.keys(memberResourceProp.attributes.properties);
+const memberResourcePath = 'members';
+const memberResourceUrl = resourcePathLink(apiBaseUrl, memberResourcePath);
+
+/**
+ * The column name getting from database is usually UPPER_CASE.
+ * This block of code is to make the camelCase keys defined in openapi.yaml be
+ * UPPER_CASE so that the serializer can correctly match the corresponding columns
+ * from the raw data rows.
+ */
+_.forEach(memberResourceKeys, (key, index) => {
+  memberResourceKeys[index] = decamelize(key).toUpperCase();
+});
```

Comment:
This function simply converts `lowerCamelCase` keys into `UPPER_SNAKE_CASE` keys. Since your openapi names and database column names don't use exactly the same names, you will need to modify this function so that the keys are correctly converted to your database column names